src/util/Constants.js

```diff
@@ -89,9 +88,9 @@ exports.UserAgent = browser ? null :
 
 exports.WSCodes = {
   1000: 'Connection gracefully closed',
```

Comment:
This is not a valid error code, destroying a Connection on startup would cause an uncaught error.
Would happen for example on an invalid token / sharding options, if there is more than one shard.